### DISCUSSION TITLE
Context menu on iOS

### DISCUSSION THREAD

**Discussion by MJegorovas** [ruffle-rs/ruffle#7712] [Q&A] [answered]
Can't seem to able to get context menu on iOS Safari (or any other browser) to show up. Trying in ruffle demo page. Other non apple devices works without a problem (long click shows the menu). Does anybody experience similar problem?
iOS 15.6 iPhone 7

**Comment by n0samu** [comment, marked as answer]
I'm pretty sure context menus on iOS have never worked and this is just a limitation of the iOS platform, unfortunately. If you have any suggestions for how Ruffle could work around this, feel free to make a feature request on the Issues tab.

**Reply by MJegorovas** [reply]
Yeah, sadly that is the [case](https://bugs.webkit.org/show_bug.cgi?id=213953). Will create an issue though and try to investigate any workarounds.

**Comment by cachepkho** [comment]
Perhaps the iOS platform's context menus do not yet enable the function, [lol beans](https://lol-beans.io)
 and this is just a restriction of the iOS platform.
